helmfile/values/fluentd-wc.yaml.gotmpl

```diff
@@ -50,7 +50,7 @@ extraConfigMaps:
     <source>
       @id kube-audit
       @type tail
-      path /var/log/kube-audit/kube-apiserver.log
+      path /var/log/kube-audit/kube-apiserver.log,/var/log/audit/kube-apiserver-audit.log
```

Comment:
Using kubespray with `audit_log_path: "/var/log/kube-audit/kube-apiserver.log"` makes the API-server Pod mount things like this:
```yaml
volumes:           
  - hostPath:                      
      path: /var/log/kubernetes/audit
...
  volumeMounts:                                                                                                                                                                                                                                                                 
    - mountPath: /var/log/kube-audit                                                                                                                                                                                                                                              
      name: audit-logs
```

So the path *inside the container* is affected, but the path on the host is actually `/var/log/kubernetes/audit/kube-apiserver.log`, which fluentd doesn't read.
After noticing this, I found another variable for kubespray: `audit_log_hostpath: /var/log/kubernetes/audit`. I think this is what we should be looking at and tell fluentd to use. See https://github.com/kubernetes-sigs/kubespray/blob/master/roles/kubernetes/master/defaults/main/main.yml#L68

Reply:
Updated fluentd path to math the default in kubespray.